client/transaction-builder/src/stdlib.rs

```diff
@@ -104,6 +104,41 @@ pub enum ScriptCall {
+    /// with this `hash` can be successfully sent to the network.
+    ///
+    /// # Parameters
+    /// | Name            | Type         | Description                                                                                     |
+    /// | ------          | ------       | -------------                                                                                   |
+    /// | `dr_account`    | `&signer`    | The signer reference of the sending account of this transaction. Must be the Diem Root signer. |
+    /// | `hash`          | `vector<u8>` | The hash of the script to be added to the script allowlist.                                     |
+    /// | `sliding_nonce` | `u64`        | The `sliding_nonce` (see: `SlidingNonce`) to be used for this transaction.                      |
+    ///
+    /// # Common Abort Conditions
+    /// | Error Category             | Error Reason                                                           | Description                                                                                |
+    /// | ----------------           | --------------                                                         | -------------                                                                              |
+    /// | `Errors::NOT_PUBLISHED`    | `SlidingNonce::ESLIDING_NONCE`                                         | A `SlidingNonce` resource is not published under `dr_account`.                             |
+    /// | `Errors::INVALID_ARGUMENT` | `SlidingNonce::ENONCE_TOO_OLD`                                         | The `sliding_nonce` is too old and it's impossible to determine if it's duplicated or not. |
+    /// | `Errors::INVALID_ARGUMENT` | `SlidingNonce::ENONCE_TOO_NEW`                                         | The `sliding_nonce` is too far in the future.                                              |
+    /// | `Errors::INVALID_ARGUMENT` | `SlidingNonce::ENONCE_ALREADY_RECORDED`                                | The `sliding_nonce` has been previously recorded.                                          |
+    /// | `Errors::REQUIRES_ADDRESS` | `CoreAddresses::EDIEM_ROOT`                                           | The sending account is not the Diem Root account.                                         |
+    /// | `Errors::REQUIRES_ROLE`    | `Roles::EDIEM_ROOT`                                                   | The sending account is not the Diem Root account.                                         |
+    /// | `Errors::INVALID_ARGUMENT` | `DiemTransactionPublishingOption::EINVALID_SCRIPT_HASH`               | The script `hash` is an invalid length.                                                    |
+    /// | `Errors::INVALID_ARGUMENT` | `DiemTransactionPublishingOption::EALLOWLIST_ALREADY_CONTAINS_SCRIPT` | The on-chain allowlist already contains the script `hash`.                                 |
+    AddToScriptAllowList { hash: Bytes, sliding_nonce: u64 },
```

Comment:
I think this change might have snuck in...I'll take a look and fix.

Reply:
Looking at this a bit more, I think this should be here...both the .mv and .abi files for this exist in the compiled transaction scripts so this should exist if it's generated from the code in upstream/master. cc @tnowacki @vgao1996 @mengxu-fb if you think that this is incorrect.

Reply:
We are killing the compiled transaction scripts for the diem framework, so I am confused about everything going on here. 

@shb has already killed the "add to allow list" functionality

Reply:
I'm not sure what's confusing, and I'm confused about why it's confusing since we haven't killed it yet?

My point/question here is that the binary for the transaction script and its abi still exist. So it should be included in the generation transaction scripts in the client, if they didn't exist, I agree and they don't need to be included. 

It's the same with the burn and cancel_burn scripts here -- the script sources don't exist anymore, but the .mv and .abi files still exist (and it was a bit painful to keep it that way...) so they should appear in the generated file for the transaction builder in order to not break clients building off of this/master today.